<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdjustmentBottom</class>
 <widget class="QWidget" name="AdjustmentBottom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>34</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>100</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AdjustBottom</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="restoreButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="dataName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="dismissButton">
     <property name="text">
      <string>Dismiss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QtAutoGui.qrc"/>
 </resources>
 <connections/>
</ui>
